### ISSUE TITLE
Inscription à "Pull Request Merge Queue Limited Beta"

### ISSUE THREAD

**Issue by adriens** [opt-nc/opt-logging#53]
# Contexte

Cf le [tweet d'annonce ce matin](https://twitter.com/GHchangelog/status/1453484477418811392), il serait très intéressant de toujours nous simplifier la gestion des PRs, en particulier sur les libs les plus utilisées et qui devraient être les plus maintenues et donc les plus maintenables.

# Détails : Why a merge queue?

Why a merge queue?

Maintaining high velocity and keeping your main branch green can be a challenge today. Many repositories try to do this by requiring all pull requests be up to date with the main branch before merging. This ensures the main branch is never updated to a commit that has not passed all required status checks, but forces developers to update (or rebase) their pull request branches multiple times and then wait for status checks to complete before trying to merge again. On some projects, status checks can take 30 or more minutes and if another pull request gets merged during this time, the process starts over (for everyone). This can have a significant impact on the overall velocity of the team and make it harder for developers to move onto their next task.

Merge Queue provides the benefits of requiring pull request branches to be up to date before merging, but without requiring developers to go through this process.

# How it works

> Merge Queue works by validating different combinations of pull requests identified as "ready to merge" in parallel so that pull requests can merge efficiently and without the typical delays that exist between merges today.

# Ressources

Détails complet de cette toute nouvelle feature : https://github.blog/changelog/2021-10-27-pull-request-merge-queue-limited-beta/

# Action

Inscrire la repo comme candidate à la **limited beta**.
